app/assets/stylesheets/local/collections.scss

```diff
@@ -0,0 +1,71 @@
+$collection-index-box-width: 266px;
+$collection-index-box-height: 266px;
+$collection-index-box-margin: 17px;
+
```

Comment:
there's probably a better way to do this CSS wihtout those ugly variables, maybe without the "spacers" -- but if this is what was ported from chf_sufia... it's not too crazy CSS relatively workable, doesn't seem necessary to refactor it now. Just wanted to point it out heh.

Reply:
Fair enough. I'm more than happy to spend some time on it of course.